Makefile

```diff
@@ -113,6 +113,9 @@ docker-build-push-tools: docker-build-tools docker-push-tools
 
 docker-build-push: docker-build-push-apiserver docker-build-push-controller
 
+swagger-ui:
+	git clone git@github.com:swagger-api/swagger-ui.git -b v2.2.10 bin/swagger-ui
```

Comment:
How about setting option `depth` to `1`?

Reply:
Good idea.